pipelines/openjdk10_nightly_pipeline.groovy

```diff
@@ -0,0 +1,42 @@
+println "building OpenJDK10"
+stage 'build OpenJDK'
+def Platforms = [:]
+Platforms["Linux"] = {
+    node {
+        build job: 'openjdk10_build_x86-64_linux', propagate: false
```

Comment:
It is not required, nor recommended to occupy a node (executor) while launching a downstream build as this will "hold" the node while the downstream job runs.

Reply:
Should that node not be held whilst the job runs though?  Not sure I follow :-)

Reply:
The downstream job will grab it's own node

Reply:
I'll confess I'm now in "I don't know how to fix this land".  I assume if I remove the node { ... } wrapper that the build job will still execute but not lock down that node?

Reply:
Correct

Reply:
Technically code not in a node block will execute on `master` but will not occupy an executor. All "real" work should always be wrapped in a node block. The part executing on master here is just the trigger of the downstream job.

Reply:
Cool, I've pushed again and removed the node blocks for non master downstream jobs